packages/flutter/test/material/tabs_test.dart

```diff
@@ -5947,6 +5947,55 @@ void main() {
     gesture.removePointer();
   });
 
+  testWidgets('Do not crash if the controller and TabBarView are updated at different phases of the same frame with SliverAppBar', (WidgetTester tester) async {
```

Comment:
Can you please make the title concise?

Reply:
Of course. However I have not seen specific suggestions for changes, so I simplified it based on myself.

```diff
-Do not crash if the controller and TabBarView are updated at different phases of the same frame with SliverAppBar
+No crash if the build of TabBar called before didUpdateWidget due to SliverAppBar
```